__tests__/components/forecast-summary.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import ForecastSummary from '../../components/forcast-summary';
+
+it('takes temperature, date, descripte and icon and displays them in spans', () => {
+  const wrapper = Enzyme.shallow(
+    <ForecastSummary
+      temperature="mockTemp"
+      date="mock date"
+      description="mock Description"
+      icon="mock icon"
+    />
+  );
```

Comment:
For the sake of thoroughness, you should probably add some assertions to check the other spans in this component. Good thinking checking the formatted text rather than just the text that is passed into it.